liquid-value/src/scalar.rs

```diff
@@ -144,6 +164,7 @@ impl<'s> ScalarCow<'s> {
             ScalarCowEnum::Integer(_) => "whole number",
             ScalarCowEnum::Float(_) => "fractional number",
             ScalarCowEnum::Bool(_) => "boolean",
+            ScalarCowEnum::DateTime(_) => "date time",
```

Comment:
`date time`, `date / time`, or something else?

This is used for things like errors.

Reply:
I think `date time` is better, since `date / time` might imply "date or time".